src/pages/header.txt

```diff
@@ -0,0 +1,10 @@
+<a href="./"> <img src="../img/logo.png" alt="politico" class="logo" /></a>
```

Comment:
I noticed you put code in a `.txt` file instead of html, is there a reason for this